.travis.yml

```diff
@@ -8,11 +8,10 @@ php:
   - nightly
 
 install:
-  - wget https://phar.phpunit.de/phpunit-old.phar
   - composer install
 
 script:
-  - php phpunit-old.phar
```

Comment:
They are equivalent ways to do the same, but I'd rather shave off the extra ~25 dependencies related to PHPUnit.

I have a strong opinion about this, config files such as phpunit.xml.dist or behat.yml are placed in the repository, but the standalone binary tools that use them must reside elsewhere.